ltfs/src/tape_drivers/osx/ltotape/ltotape_supdevs.h

```diff
@@ -43,6 +43,7 @@ const supported_device_type supported_devices[] = {
    { "ULTRIUM 7       ", drivefamily_lto,     "Quantum LTO7",      	drive_lto7		},
    { "LTO-5 HH        ", drivefamily_lto,     "TANDBERG DATA LTO5", drive_lto5 		},
    { "LTO-6 HH        ", drivefamily_lto,     "TANDBERG DATA LTO6",	drive_lto6		},
+   { "ULTRIUM-HH5     ", drivefamily_lto,     "DELL LTO5-140",      drive_lto5		},
```

Comment:
@sardaukar On our setting (Ubuntu and Slackware with HP 6250), yes. And we are working on Windows as well.